tests/e2e/lib/backup.go

```diff
@@ -145,3 +145,50 @@ func BackupErrorLogs(c *kubernetes.Clientset, ocClient client.Client, namespace
 	bl := BackupLogs(c, ocClient, namespace, name)
 	return errorLogsExcludingIgnored(bl)
 }
+
+func GetBackupRepositoryList(c client.Client, namespace string) (*velero.BackupRepositoryList, error) {
+	backupRepositoryList := &velero.BackupRepositoryList{
+		Items: []velero.BackupRepository{},
+	}
+	err := c.List(context.Background(), backupRepositoryList, client.InNamespace(namespace))
+	// if there is an error, just print, don't return the error
```

Comment:
I think this is anti pattern in GO

would return nil, error here

Reply:
ok, I agree.  Fixed.

Reply:
ur right, fixed